torchbearer/callbacks/torch_scheduler.py

```diff
@@ -12,32 +12,53 @@ def __init__(self, scheduler_builder, monitor=None, step_on_batch=False):
         self._scheduler = None
         self._step_on_batch = step_on_batch
 
+        from distutils.version import LooseVersion
+        version = torch.__version__ if str(torch.__version__) is torch.__version__ else "0.4.0"
+        self._newstyle = LooseVersion(version) > LooseVersion("1.1.0")
+
+    def _step(self, state, current=None):
+        if self._newstyle:
```

Comment:
```suggestion
        if self._newstyle or self._step_on_batch:
```
Don't need to pass epoch argument if stepping on batch (since it would then turn into a step argument anyway)